README.md

```diff
@@ -1,7 +1,242 @@
 # Transistor
 
-A Crux Client crate.
+A Crux Client crate. For now, this crate intends to support 2 ways to interact with Crux:
 
-For examples on usage, please refer to [examples directory](https://github.com/naomijub/transistor/tree/master/examples), and for information on crux and how to use it, please follow the link to [opencrux](https://opencrux.com/docs#restapi)
+* Via `Docker` with a [`crux-standalone`](https://opencrux.com/docs#config-docker) version [docker-hub](https://hub.docker.com/r/juxt/crux-standalone).
```

Comment:
What do you think of those two items being checkboxes? Being the first one checked and the second one unchecked.